UI/SitRepPanel.cpp

```diff
@@ -269,7 +265,7 @@ namespace {
             AttachChild(m_icon);
 
             m_link_text = new LinkText(GG::X0, GG::Y0, GG::X1, m_sitrep_entry.GetText() + " ", 
-                                       ClientUI::GetFont(std::max(1.0*ClientUI::Pts(), 1.0*GetOptionsDB().Get<int>("UI.sitrep-font-size"))),
+                                       ClientUI::GetFont(ClientUI::Pts()),
```

Comment:
The `1.0*` directly before `ClientUI::Pts()` was still there and has been removed. But I guess what you meant is that `ClientUI::Pts()` can be removed as well, as `ClientUI::GetFont()` without any parameters will yield the exact same result anyway, correct?